packages/components/pie-divider/src/divider.scss

```diff
@@ -1,22 +1,39 @@
 @use '@justeattakeaway/pie-css/scss' as p;
 
 .c-divider {
-    --divider-bg-color: var(--dt-color-divider-default);
-    --divider-width: 100%;
-    --divider-height: 1px;
+  --divider-bg-color: var(--dt-color-divider-default);
+  --divider-width: 100%;
+  --divider-height: 1px;
 
+  display: flex;
+  align-items: center;
+  text-align: center;
+  height: var(--divider-height);
```

Comment:
We should probably not be setting the height to 1px if we're adding text:
 
<img width="1455" alt="Screenshot 2024-08-29 at 15 57 55" src="https://github.com/user-attachments/assets/c1b2e7f4-fe22-43e4-ac30-221fcdf293be">